x/mint/keeper.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/params"
 )
 
+var minterKey = []byte{0x00} // the one key to use for the keeper store
+
 const (
 	// ModuleName is the name of the module
 	ModuleName = "minting"
```

Comment:
this differs with the module name (?)

Reply:
indeed. I didn't write this module -- it should be changed. Separate PR though.